integrations/server/test_covidcast.py

```diff
@@ -194,7 +193,7 @@ def test_fields(self):
     row = self._insert_placeholder_set_one()
 
     # limit fields
-    response = self.request_based_on_row(row, fields='time_value,geo_value')
+    response = self.request_based_on_row(row, **{"fields":"time_value,geo_value"})
```

Comment:
question: did you ever figure out why the **{} is needed?

it's fine this way, just curious

Reply:
Both ways work now. I choose to keep it like this to be consistent with how extra arguments are added in other test functions, but having it as fields='time_value,geo_value' also works.